hudi-sync/hudi-adb-sync/src/main/java/org/apache/hudi/sync/adb/AdbSyncTool.java

```diff
@@ -189,6 +190,13 @@ private void syncHoodieTable(String tableName, boolean useRealtimeInputFormat, b
     }
     LOG.info("Last commit time synced was found:{}", lastCommitTimeSynced.orElse("null"));
 
+    String currentInstantTs = config.getString(META_SYNC_CURRENT_INSTANT_TS);
+    if (lastCommitTimeSynced.isPresent() && !currentInstantTs.isEmpty()) {
+      // If occ is enabled, the commit time may be earlier than lastCommitTimeSynced.
+      // In this case, the metadata of the current instant cannot be synchronized, resulting in the loss of partition metadata.
+      lastCommitTimeSynced = currentInstantTs.compareTo(lastCommitTimeSynced.get()) < 0 ? Option.of(currentInstantTs) : lastCommitTimeSynced;
+      LOG.info("Reset lastCommitTimeSynced " + lastCommitTimeSynced.get());
```

Comment:
Somehow we need to rememer the client sync time for each writer, or keep all the instant times that was synced in server, the timestamp comparison is tricky and likely to produce bugs.

Reply:
@danny0405  danny, There are two problems:
1. I am a little confused about why the use of timestamp comparison causes bugs
2. Any suggestions if you want to rememer instant timestamp